src/coreclr/vm/common.h

```diff
@@ -128,7 +128,7 @@ typedef DPTR(class EEConfig)            PTR_EEConfig;
 typedef VPTR(class EEDbgInterfaceImpl)  PTR_EEDbgInterfaceImpl;
 typedef VPTR(class DebugInfoManager)    PTR_DebugInfoManager;
 typedef DPTR(class FieldDesc)           PTR_FieldDesc;
-typedef VPTR(class Frame)               PTR_Frame;
+typedef DPTR(class Frame)               PTR_Frame;
```

Comment:
Do we need to review `(SomeFrame*)` casts in DACized code like this one:
https://github.com/dotnet/runtime/blob/4dcd4d37fb65f10ce1d1ea370a11a7bf55155a40/src/coreclr/debug/daccess/dacdbiimplstackwalk.cpp#L521 ?

This cast works now since DAC magic will get the actual type of the instance and copy over all memory that the instance needs. It is no longer going to be the case with DPTR pointer.

Reply:
Sigh. Yes. This is reason 72947294 why the DAC is not my friend.

Reply:
My fixes to this have made the debugger happy.